evergreen/remove-certs.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+if [[ "$OS" =~ Windows|windows ]]; then
```

Comment:
We need the case-insensitive variant because we override the $OS variable in evergreen.yml at one point and set it to windows-64.
https://github.com/vincentkam/mongo-csharp-driver/blob/c084d08e28acdc78a60ebb401c99a48d8033b1a9/evergreen/evergreen.yml#L403